packages/tools/locales/nl_NL.po

```diff
@@ -3302,9 +3294,8 @@ msgid "Downloaded and encrypted"
 msgstr "Gedownload en versleuteld"
 
 #: packages/lib/services/ReportService.js:180
-#, fuzzy
 msgid "Created locally"
-msgstr "Lokale items gecreëerd: %d."
+msgstr "Lokale items gecreëerd: %d"
```

Comment:
This translation does not seem to be correct. `Lokale items gecreëerd: %d` is already used on line 2988 and is the translation for `Created local items: %d.`.